gwlfe/CalcCnErosRunoffSed.py

```diff
@@ -163,8 +163,8 @@ def CalcCN(z, i, Y, j):
             if z.Water >= 0.2 * z.CNumPervReten:
                 z.QrunP[l] = (z.Water - 0.2 * z.CNumPervReten) ** 2 / (z.Water + 0.8 * z.CNumPervReten)
 
-        # TODO: Should 11 be NRur + 1? What is this trying to do?
-        lu = l - 11
+        # TODO: What is this trying to do?
+        lu = l - 10
```

Comment:
I asked for clarification on `11` and if `NRur` was appropriate.  The response was there might be a bug in the original, so I'm going to hold off until I hear back. :confused: